### ISSUE TITLE
shell client does not handle errors

### ISSUE THREAD

**Issue by javiplx** [javiplx/ameba-C3#10]
Error conditions are not properly handled.
If a server error happens processing the request, the session below is produced

```
root@MeshWeb30:/tmp# ./ameba-updater.sh -r -m avgload,netstat,test,meminfo-24 register http://r26936.ovh.net/amebaC3
wget: server returned error: HTTP/1.1 500 Internal Server Error
sed: /tmp/aupd.response.4086: No such file or directory
grep: /tmp/aupd.response.4086: No such file or directory
 root@MeshWeb30:/tmp# 
```

**Comment by javiplx** [comment]
Same behaviour is found in timeout also

**Comment by javiplx** [comment]
And another more exotic case

```
root@MeshWeb30:~# /bin/ameba-updater.sh -w 1 pull
wget: server returned error: HTTP/1.1 500 Internal Server Error
wget: server returned error: HTTP/1.1 401 Authorization Required
```

**Comment by javiplx** [comment]
Fixed on commit 9db0cc23e7f21ca8ff5492c18c81d721cda3d0b9
